torch/quantization/fx/utils.py

```diff
@@ -94,23 +94,30 @@ def get_per_tensor_qparams(activation_post_process):
     dtype = activation_post_process.dtype
     return scale, zero_point, dtype
 
-def get_quantize_op_and_qparams(activation_post_process):
+def get_quantize_node_info(activation_post_process):
```

Comment:
any chance we can annotate the return type?  Would be great for readability, especially since we are changing the return type

Reply:
sure